docs/community/README.md

```diff
@@ -0,0 +1,14 @@
+# Community
+
+Here you'll find documentation geared towards leaning about the development process for the AKS-Engine project itself and how you can contribute.
+
+- [Code of Conduct](code-of-conduct.md)
+- [Developer Guide](developer-guide.md)
+- [Planning Process](planning-process.md)
+- [Release Checklist](release-checklist.md)
+
+AKS-Engine is a community effort. As it keeps growing, we always need more people to help others. As soon as you learn AKS-Engine, you can contribute in many ways:
+
+- Join the #sig-azure Slack channel on <https://kubernetes.slack.com> and answer questions. By explaining AKS-Engine to other users, you’re going to learn a lot about the tool yourself.
+- Blog about AKS-Engine. We syndicate all the AKS-Engine blogs we know about on the [topics page](../topics/README.md); if you’d like to see your blog on that page, you are more than welcome to add it there.
+- Contribute to open-source AKS-Engine projects, write some documentation, or release your own code as an open-source extension. The ecosystem of extensions is a community effort; help us build it!

```

Comment:
`open-source AKS-Engine projects` ?

Reply:
third-party projects that vendor aks-engine as a dependency, either the SDK itself or the binary. Know of any?

Reply:
`Contribute to other projects that use aks-engine, write documentation, or release your own code as an open-source extension. The ecosystem of extensions is a community effort; help us build it!`

How about this ?

Reply:
@bacongobbler I don't know of any.